### PR TITLE
Feature/saml2 login

### LINKED ISSUE #57: Allow users to log in via SAML2 identity provider

### PR DESCRIPTION
**Pull request by jykae**
Closes #57 

# Proposed changes
- Remove force-ssl (based on common discussion, use nginx frontend instead)
- Add fixed/patched apinf:meteor-accounts-saml package
- Ignore saml settings (Be careful here..)
- Provide example settings TODO

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by jykae** [comment]
Steps:
- making final package changes for login handling, verify reasonable security
- make settings changes according final changes if any

File: saml/client/login.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<template name="samlLogin">
+  <p>
+    <a href="#" class="saml-login" data-provider="mpass">Login with MPASS</a>
```
Comment (brylie) [inline saml/client/login.html]: Remember to add i18n tokens here. E.g. 

```
{{_ "samlLogin-mpass-button" }}
```

File: saml/client/login.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  </p>
+  <p>
+    <a href="#" class="saml-login" data-provider="eduuni">Login with eDuuni</a>
```
Comment (brylie) [inline saml/client/login.html]: Remember to add i18n tokens here. E.g. 

```
{{_ "samlLogin-edunni-button" }}
```

**Comment by brylie** [comment]
@jykae what is the status of this PR?

**Comment by jykae** [comment]
Closed because security issues with xml-crypto.

**Pull request closed without merging**